### DISCUSSION TITLE
Do we have presto native worker deployment guide?

### DISCUSSION THREAD

**Discussion by xumingming** [facebookincubator/velox#7177] [Q&A] [answered]
I know there is a DEVELOPMENT guide for presto native worker(https://github.com/prestodb/presto/tree/master/presto-native-execution), do we have DEPLOYMENT guide for deploying presto native worker in linux production environment?

**Comment by mbasmanova** [comment]
CC: @aditi-pandit @majetideepak @amitkdutta Folks, do you have anything to share?

**Comment by aditi-pandit** [comment, marked as answer]
@mbasmanova : 
This discussion is cross-posted in Presto github repo as well. https://github.com/prestodb/presto/issues/21242
That is a better forum for this topic I think. Closing the discussion here.
